CONTRIBUTING.md

```diff
@@ -31,106 +41,112 @@ To send us a pull request, please:
 
 1. Fork the repository.
 2. Modify the source; please focus on the specific change you are contributing. If you also reformat all the code, it will be hard for us to focus on your change.
-3. Ensure local tests pass.
+3. Ensure local tests pass (check the [scripts](https://github.com/aws-samples/aws-cdk-examples/tree/master/scripts) directory)
 4. Commit to your fork using clear commit messages.
 5. Send us a pull request, answering any default questions in the pull request interface.
 6. Pay attention to any automated CI failures reported in the pull request, and stay involved in the conversation.
 
 GitHub provides additional document on [forking a repository](https://help.github.com/articles/fork-a-repo/) and
 [creating a pull request](https://help.github.com/articles/creating-a-pull-request/).
 
-## Adding an example
+## Adding an Example <a name="New"></a>
 
-Create a directory with a name that is descriptive of the resource type or workflow you are trying to create, e.g. ecs-load-balanced-service. You will need to create the following four files in your example:
-1. `index.ts`, containing your CDK application code.
-2. `cdk.json`, containing:
+When adding a new example, there are several things important to consider as you implement (please create a `feature-request` issue to discuss these before writing your contribution):
 
-```
-{
-    "app": "node index"
-}
-```
+### Added Value
+As an official learning resource, it is important that any new examples add value to our learning resources. This means we are looking for examples that clarify additional information about developing with the CDK whether this is a common pattern, or a construct library that does not have any examples.
 
-3. `tsconfig.json`, containing:
+When considering if your example adds value, review the existing examples and think about if your example could be written with just the knowledge from the other examples.
 
-```
-{
-    "compilerOptions": {
-        "target":"ES2018",
-        "module": "commonjs",
-        "lib": ["es2016", "es2017.object", "es2017.string"],
-        "strict": true,
-        "noImplicitAny": true,
-        "strictNullChecks": true,
-        "noImplicitThis": true,
-        "alwaysStrict": true,
-        "noUnusedLocals": true,
-        "noUnusedParameters": true,
-        "noImplicitReturns": true,
-        "noFallthroughCasesInSwitch": false,
-        "inlineSourceMap": true,
-        "inlineSources": true,
-        "experimentalDecorators": true,
-        "strictPropertyInitialization":false
-    }
-}
-```
+### Language Parity
+While we have historically allowed adding examples in just one of our supported languages, moving forward we would like to maintain language parity. This does not necessarily mean that you must write the example in each action yourself, but a PR will not be merged until the example has been added for each language.
```

Comment:
"in each action", do you mean "each language"? Also I might, list out all of our languages as a reminder.

Reply:
Good catch on "each language". I am hesitant to list the languages because of the flux as we add more.

Reply:
```suggestion
While we have historically allowed adding examples in just one of our supported languages, moving forward we would like to maintain language parity. This does not necessarily mean that you must write the example in each language yourself, but a PR will not be merged until the example has been added for each language.
```